lib/std/fmt.zig

```diff
@@ -1,3 +1,5 @@
+thisfileisautotranslatedfromc;
```

Comment:
Maybe this has been discussed already, but for the uncommon correct uses of C pointers in std (testing functionality that interacts with them), how about using `@Type` to programmatically construct C pointer types? Since the check is implemented in AstGen, I believe that should still work, and it would avoid having to infect the entire file with the autotranslated status.

Reply:
This was discussed, and that was the consensus iirc.
If you look at my checklist I still have to look through the stdlib uses of the keyword and see if they are truly needed. I will construct with with `@Type` if they actually are needed 👍